Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -860,13 +863,14 @@ def get_store_data(service):
         store = service.kvstore[store]
         query = build_kv_store_query(store, args)
         if 'limit' not in query:
-            # query = {'query': json.dumps({key: val for key, val in json.loads(query).items()})}
-            # query = {'query': json.dumps(json.loads(query))}
             query = {'query': query}
         yield store.data.query(**query)
 
 
 def main():
+    if demisto.command() == 'splunk-parse-raw':
+        splunk_parse_raw_command()
+        sys.exit(0)
```

Comment:
why?

Reply:
we don’t need to configure a splunk client.